effekt/jvm/src/main/scala/effekt/Runner.scala

```diff
@@ -78,6 +84,25 @@ trait Runner[Executable] {
     }
     go(progs0)
   }
+
+  /**
+   * Create a executable file with at the given path with the given content. The file will have
+   * the permissions UNIX permissions 744.
+   */
+  def createExecutableFile(path: String, content: String)(using C: Context): Unit = {
+    import java.nio.file.{Files, Path}
+    import java.nio.file.attribute.PosixFilePermission.*
+    import scala.jdk.CollectionConverters.SetHasAsJava
+
+    val path1 = Path.of(path)
+    val perms = Set(
+      OWNER_READ, OWNER_WRITE, OWNER_EXECUTE,
+      GROUP_READ,
+      OTHERS_READ
+    )
+    IO.createFile(path, content)
+    Files.setPosixFilePermissions(path1, SetHasAsJava(perms).asJava)
```

Comment:
Maybe this could be used to our kiama fork as a helper function `makeExecutable` or similar?